extjsdk/src/test/java/io/vantiq/extjsdk/RoundTripTestBase.java

```diff
@@ -40,6 +41,8 @@ public static void getProps() {
         testSourceName = System.getProperty("EntConTestSourceName", "testSourceName");
         testTypeName = System.getProperty("EntConTestTypeName", "testTypeName");
         testRuleName = System.getProperty("EntConTestRuleName", "testRuleName");
+        testRepeatedConnectsEnabled =
+                Boolean.valueOf(System.getProperty("TestRepeatedConnects", "false"));
```

Comment:
Temporary -- we can remove when we're testing mostly against 1.32+ systems

Reply:
We should probably add an issue reminding us to remove this once 1.32 is out.